app/services/data_export/content_export.rb

```diff
@@ -98,8 +98,20 @@ def get_content(base_path, base_fields: CONTENT_BASE_FIELDS, taxon_fields: CONTE
       {}
     end
 
+    def blacklisted_content_stats(document_types = ContentExport::BLACKLIST_DOCUMENT_TYPES)
+      results = document_types.map do |document_type|
+        { document_type: document_type,
+          count: number_of_documents(document_type) }
+      end
+      results.sort_by {|r| -r[:count]}
+    end
+
   private
 
+    def number_of_documents(document_type)
+      Services.rummager.search(filter_content_store_document_type: [document_type], count: 0)['total']
```

Comment:
Have you seen the aggregation features in the Search API? It might be quicker and simpler to make one request for all of the data (e.g. [1]), and then pull out the blacklisted document types.

1: https://www.gov.uk/api/search.json?count=0&aggregate_content_store_document_type=1000